securemeet.py

```
@@ -0,0 +1,986 @@
+            }
+        try:
+            tmp = tempfile.NamedTemporaryFile(suffix=".wav", delete=False)
+            sf.write(tmp.name, audio_data, sample_rate)
+            diar = self.pipeline(tmp.name)
+            os.unlink(tmp.name)
+            segs, spks = [], set()
+            for turn, _, sp in diar.itertracks(yield_label=True):
+                segs.append({"start": turn.start, "end": turn.end, "speaker": sp})
+                spks.add(sp)
+            return {"speakers": list(spks), "segments": segs}
+        except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)